xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/register_macros.vm

```diff
@@ -19,7 +19,7 @@
 ## ---------------------------------------------------------------------------
 ## Defines what server generated error messages should look like
 ## The error message when a field is entered incorrectly
-#set ($failureMessageParams = {'class': 'LV_validation_message LV_invalid'})
+#set ($failureMessageParams = {'class': 'LV_invalid'})
```

Comment:
Why making that change? And you're sure it doesn't impact any test?

Reply:
The way I use this variable changed. I removed this class because it's a generic class and it's not specific to the failureMessage.

See https://github.com/xwiki/xwiki-platform/blob/683ccd55027e46a1f7f86a8aaa6f17186fc2745d/xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/register_macros.vm#L176-L180

> And you're sure it doesn't impact any test?

To be honest I didn't remember what I did exactly to check, but I did check tests to make sure they'd still pass. I brought changes to RegisterIT and RegistrationPage to make sure they'd still work properly.
See [the top message with details about the tests I did](https://github.com/xwiki/xwiki-platform/pull/3155#issue-2336273733)